### DISCUSSION TITLE
Background Informational Page for App

### DISCUSSION THREAD

**Discussion by rory-spurr** [rory-spurr/ESAPermitsCapstone#77] [Show and tell]
This is a place to discuss the layout/design of the informational page I have drafted. I intend for this discussion page to be the more "big picture" discussions (i.e. bulleted list structure, spacing between paragraphs, txt size etc.) and I have also created a 
[google doc](https://docs.google.com/document/d/17mPG5k8kQKWYYv3p8AMjJXEVMbkb_LB94G8bw_2783g/edit) where we can sequester the more smaller scope edits (word usage, sentence structure, jargon etc.). 

Apologies for having the edits in multiple places, but I found this to be the best way for everyone to contribute to both the larger and smaller scope edits simultaneously.

**Comment by DianaDishman-NOAA** [comment]
I know I should have brought this up earlier, but is there any way to switch which elements are on the tabs and sidebar menu? So that the Tabs across the top would be "Home," "Map," and "Plots," and then when you're on the "Home" tab you would see a left sidebar of "Background," "How it Works," and "Disclaimer"?  If you run the app and it opens in a narrow browser window, or if you're default is to close the menu, you can only see the content of the 'Home' tab but you don't know that the menu will lead you to the map and plots. This could just be personal preference, but I think it would be a good idea if the Map and Plots sections are *always* visible, no matter how you configure viewing setup.

**Reply by DianaDishman-NOAA** [reply]
To address this we decided on instead making all of the tab content part of the drop down menu on the left side, and not allowing the left sidebar menu be collapsible.

**Comment by annebeau** [comment]
I recommend including a 'Glossary' section under the Home menu that defines all of the terms you'll be using throughout the text. That way, you don't have to keep defining them and can direct readers to the glossary.

**Reply by asantan8** [reply]
Would it be a good idea to make the glossary into a table that way users can search for specific words? Or should we keep our glossary section short?

**Comment by annebeau** [comment]
The large legend under the 'Build Your Plot' section is not needed (and is slightly distorted). I suggest moving the "Getting Started" text to the side, either above or below the "Build Your Plot" menus. That will fill the space and will also ensure that the app user sees the text description.

**Comment by annebeau** [comment]
I recommend enlarging all of the text within the app (e.g., in the narrative sections under Home, in the 'Getting Started' section of the time series plots, etc.)

**Reply by rory-spurr** [reply]
Good point! I am going to make this an issue.
